src/lib/flow/Kconfig

```diff
@@ -4,8 +4,60 @@ config FLOW
 
 config RESOLVER_CONFFILE
 	bool "Resolver conffile"
+	depends on FLOW
 	default y
 
-config FLOW_RESOLVER
-	bool "Flow resolver"
+config NODE_DESCRIPTION
+	bool "Node description support"
+	depends on RESOLVER_CONFFILE
```

Comment:
This is not right, one can have NODE_DESCRIPTION without a conffile resolver (i.e.: one can provide one externally in his binary)

OTOH, RESOLVER_CONFFILE needs NODE_DESCRIPTION.